environs/imagedownloads/simplestreams.go

```diff
@@ -99,18 +99,21 @@ func Fetch(
 			ValueTemplate: Metadata{},
 		},
 	}
-	items, resolveInfo, err := simplestreams.GetMetadata(src, params)
+	results, err := simplestreams.GetMetadata(src, params)
 	if err != nil {
-		return nil, resolveInfo, err
+		return nil, nil, err
 	}
-	md := make([]*Metadata, len(items))
-	for i, im := range items {
+	if len(results) < 1 {
+		return nil, nil, errors.NotFoundf("simplestreams metadata results")
+	}
+	md := make([]*Metadata, len(results[0].Items))
```

Comment:
Why results[0]?
We should be iterating over all returned results

Reply:
In image downloads? We'll never pass UseAllSources=true here, why iterate through the results?